src/app/components/layouts/card/terra-card.component.spec.ts

```diff
@@ -1,3 +1,198 @@
-/**
- * @author michaelkunze
- */
+import {
+    Component,
+    DebugElement
+} from '@angular/core';
+import {
+    async,
+    ComponentFixture,
+    TestBed
+} from '@angular/core/testing';
+import { TerraCardComponent } from './terra-card.component';
+import { By } from '@angular/platform-browser';
+
+@Component({
+    template: `
+                  <terra-card>
+                      <div terra-card-header>
+                          <p>card header</p>
+                      </div>
+                      <div terra-card-content>
+                          <p>card content</p>
+                      </div>
+                      <div terra-card-footer>
+                          <p>card footer</p>
+                      </div>
+                  </terra-card>
+              `
+})
+class CardTestComponent{}
+
+describe('TerraCardComponent', () =>
+{
+    let component:CardTestComponent;
+    let cardComponent:TerraCardComponent;
+    let fixture:ComponentFixture<CardTestComponent>;
+    let debugElement:DebugElement;
+
+    const expectedImagePath:string = 'app/assets/images/logo_plenty.svg';
+    const expectedIcon:string = 'icon-save';
+
+    beforeEach(async(() =>
+    {
+        TestBed.configureTestingModule({
+            declarations: [TerraCardComponent,
+                           CardTestComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() =>
+    {
+        fixture = TestBed.createComponent(CardTestComponent);
+        component = fixture.componentInstance;
+        cardComponent = fixture.debugElement.query(By.css('terra-card')).componentInstance;
+        fixture.detectChanges();
+        debugElement = fixture.debugElement;
+    });
+
+    afterEach(() =>
+    {
+        cardComponent.inputPlaceholderIcon = null;
+        cardComponent.inputImagePath = null;
+    });
+
+    describe('footer content', footer);
+    describe('header content', header);
+    describe('block content', block);
+    describe('image', image);
```

Comment:
Diese Tests gerne nach 'should create' und 'should initialize' platzieren

Reply:
done